### ISSUE TITLE
Datawrapper Symbol map not rendering in Pageflow

### ISSUE THREAD

**Issue by RoOnFire** [codevise/pageflow#1731]
Hello dear Pageflow Team,

an external service contractor contacted due to a map type of datawrapper not properly working on their pageflow hosted by us.

All map types are being rendered correctly, as of the given information i have received by the contractor, except the "Symbol map". Is there a parsing error for this datawrapper map?

There are no errors in the log files but i see the following error response in my Chrome DevTools when inserting the map:

```html
<Error
>
<Code
>
NoSuchKey
</Code
>
<Message
>
The specified key does not exist.
</Message
>
<Key
>
main/pageflow/chart/scraped_sites/000/lib/basemaps/germany-states.60616814.json
</Key
>
<RequestId
>
R168WN743PHH9HMJ
</RequestId
>
<HostId
>
makSyOTnX6nxpEGBcVlAeMaNYcVftds2OrY4rIxwEwwR7DQyIUUivnLTn58RgJNDr8gx1bEHw/E=
</HostId
>
</Error>
```

The Request shows a 404 Status Code
The Request URL ist the following (without domain) :
/datawrapper/main/pageflow/chart/scraped_sites/000/lib/basemaps/germany-states.60616814.json

Thanks in advance and best regards,
Ronald

**Comment by tf** [comment]
We also encountered problems recently with new Datawrapper chart types that apparently load additional data via XHR. These JSON files are not included when we scrape the charts and are thus missing when charts are displayed in Pageflow. Since it will be very hard to predict which files actually need to be scraped, we currently consider moving away from the scraping approach. Since we still need to serve the charts from same origin to allow for some more advanced JavaScript interaction between chart and outer page, we consider adapting the recommended proxy setup to point to the original Datawrapper CDN endpoints instead of the S3 bucket with the scraped charts. This is still work in progress, though.

As a workaround, we found that the affected charts mostly load JSON files from relative URLs of the form ../../whatever.json. We've been able to fix individual charts by manually placing required JSON files in the corresponding directory on S3. This is of course a terrible hack, but can help out customers that just need a specific chart to work.
